testing_and_setup/compass/ocean/drying_slope/forward_template.xml

```diff
@@ -26,7 +27,7 @@
 		<option name="config_use_variable_drag">.true.</option>
 		<option name="config_Rayleigh_friction">.true.</option>
 		<option name="config_Rayleigh_damping_depth_variable">.true.</option>
```

Comment:
@caozd999, we want this to be a ridiculous init value so that it breaks if not specified in a test case.  Is there a reason to set this to something particular?

Reply:
You are right, there is no reason to change this value.